arsceneview/src/main/kotlin/io/github/sceneview/ar/interaction/Translation.kt

```diff
@@ -28,25 +26,13 @@ internal class TranslationGesture(arNode: ArNode) : GestureStrategy(arNode) {
     }
```

Comment:
And why limiting to planes? Others (depth and instant placement) might also be used.

Reply:
The idea was to just use it as a filter for plane because they're the only ones were you would need to differentiate between vertical and horizontal ones. But I guess somebody could (and should) always configure this behavior using the Config.planeFindingMode . So I will remove the filter to simplify the code.